.github/workflows/preview.yml

```diff
@@ -0,0 +1,51 @@
+name: Upload Preview
+
+on:
+  workflow_run:
+    workflows: ["Build Preview"]
+    types:
+      - completed
+
+jobs:
+  upload:
+    name: 'upload PR preview'
+    runs-on: ubuntu-latest
+    if: >
+      ${{
+        github.event.workflow_run.event == 'pull_request' &&
+        github.event.workflow_run.conclusion == 'success' &&
+        github.repository == 'tc39/ecma262'
+      }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ljharb/actions/node/install@56f291b2a0d9e3bce72634356ae949b1e054b9f8
+        name: 'nvm install lts/* && npm ci'
+        with:
+          node-version: lts/*
+          use-npm-ci: true
+      - name: 'Download artifact'
+        uses: actions/github-script@v3.1.0
+        with:
+          script: |
+            const { owner, repo } = context.repo;
+            const artifacts = await github.actions.listWorkflowRunArtifacts({
+               owner,
+               repo,
+               run_id: ${{ github.event.workflow_run.id }},
+            });
+            const [{ id: artifact_id }] = artifacts.data.artifacts.filter((artifact) => artifact.name == 'out');
```

Comment:
Random nit:

```suggestion
            const { id: artifact_id } = artifacts.data.artifacts.find((artifact) => artifact.name == 'out');
```

Reply:
ha, thanks. this was tweaked copy-pasta; i'll pull in your suggestion.